sdk/python/feast/permissions/auth/kubernetes_token_parser.py

```diff
@@ -41,10 +42,14 @@ async def user_details_from_access_token(self, access_token: str) -> User:
         current_user = f"{sa_namespace}:{sa_name}"
         logging.info(f"Received request from {sa_name} in {sa_namespace}")
 
-        roles = self.get_roles(sa_namespace, sa_name)
-        logging.info(f"SA roles are: {roles}")
+        skip_auth_key = os.getenv("SKIP_AUTH_KEY")
```

Comment:
Not clean who and when this env var will be set.
I would suggest from the server's init block (maybe the `init_auth_manager` function?)

Reply:
Why not using the helm chart as Abdul mentioned, if its not set in the code it allows more flexibility in case the value is compromised it can be changed with no restart for example.

Reply:
it also works, of course.